olm-catalog/serverless-operator/project.yaml

```diff
@@ -6,7 +6,7 @@ project:
     # Also make sure to update values under `olm.previous` by copying from `olm.replaces` and `olm.skipRange`.
     version: 1.32.0
 olm:
-    replaces: 1.31.0
+    replaces: 1.31.1
```

Comment:
I am not really sure we need this.  as we have a range?

Usually it would be ok if something _like_ 1.31.1 comes out **_AFTER_** the 1.32 (think LTS)

Reply:
Right.

Reply:
/close 
as discussed in the comments above
this should _NOT_ be needed, at all